plugins/main/windows/win-overlay/win-overlay_windows.go

```diff
@@ -173,5 +173,5 @@ func cmdCheck(_ *skel.CmdArgs) error {
 }
 
 func main() {
-	skel.PluginMain(cmdAdd, cmdCheck, cmdDel, version.PluginSupports("0.1.0", "0.2.0", "0.3.0"), bv.BuildString("win-overlay"))
+	skel.PluginMain(cmdAdd, cmdCheck, cmdDel, version.PluginSupports("0.1.0", "0.2.0", "0.3.0", version.Current()), bv.BuildString("win-overlay"))
```

Comment:
This should have "0.4.0" in it as well.

Reply:
also `version.All`

Reply:
nevermind, this comment is wrong.

Reply:
this should be `version.All` rather than a list.

Reply:
fixed